docs/source/overview/prerequisites.rst

```diff
@@ -86,7 +96,29 @@ An example is the following:
     - 2.97384953722449e-05
     - 4.511438394964044e-05
     - 6.843744918967896e-05
-    ...
+    - 0.00010381172986576898
+    - 0.00015745605600841445
+    - 0.00023878782918561914
+    - 0.00036205449638139736
+    - 0.0005487795323670796
+    - 0.0008314068836488144
+    - 0.0012586797144272762
+    - 0.0019034634022867384
+    - 0.0028738675812817515
+    - 0.004328500638820811
+    - 0.006496206194633799
+    - 0.009699159574043398
+    - 0.014375068581090129
+    - 0.02108918668378717
+    - 0.030521584007828916
+    - 0.04341491741702269
+    - 0.060480028754447364
+    - 0.08228122126204893
+    - 0.10914375746330703
+    - 0.14112080644440345
+    - 0.17802566042569432
+    - 0.2195041265003886
+    - 0.2651137041582823
```

Comment:
Please, keep the ellipsis. It makes the page much more readable, and it is fairly simple to infer how to complete.

If you really wish, attach the actual runcard, but do not increase the length of the one displayed.

Reply:
[024c6d9](https://github.com/NNPDF/pineko/pull/98/commits/024c6d96f13a209a155f5af0240f6bbefae57642)